babblesim/apps/edtt_test/hci_test_app/syscfg.yml

```diff
@@ -0,0 +1,31 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+syscfg.vals:
+    BLE_HOST: 0
+    BLE_HCI_TRANSPORT: custom
+    MCU_UART_POLLER_PRIO: 3
```

Comment:
I think we could just change uart poller prio in bsim bsp to smth like 9, I assume it should have lowest priority of all tasks. Then edtt poller can be set to 2 (as it is now) and there's no need to override uart.